src/postgraphile/http/createPostGraphileHttpRequestHandler.ts

```diff
@@ -413,7 +413,11 @@ export default function createPostGraphileHttpRequestHandler(
         )
       : null;
 
-    if (subscriptions) {
+    if (
+      subscriptions ||
+      options.websocketOperations == 'all' ||
+      options.websocketOperations == 'subscriptions'
```

Comment:
The subscriptions flag also needs to be set, I think - other things depend on it.

```suggestion
      subscriptions && (
        options.websocketOperations === 'all' ||
        options.websocketOperations === 'subscriptions'
      )
```